### ISSUE TITLE
ClassCastException: com.softw4re.infinitelistview.MyInfiniteListAdapter cannot be cast to android.widget.HeaderViewListAdapter

### ISSUE THREAD

**Issue by aitbaali** [ugurcany/InfiniteListView#6]
Run The Example Project

```
FATAL EXCEPTION: main 
java.lang.ClassCastException: com.softw4re.infinitelistview.MyInfiniteListAdapter cannot be cast to android.widget.HeaderViewListAdapter
at android.widget.ListView.removeFooterView(ListView.java:387)
at com.softw4re.views.InfiniteListView.stopLoading(InfiniteListView.java:167)
at com.softw4re.infinitelistview.MainActivity$1.onPostExecute(MainActivity.java:79)
at com.softw4re.infinitelistview.MainActivity$1.onPostExecute(MainActivity.java:46)
at android.os.AsyncTask.finish(AsyncTask.java:631)
at android.os.AsyncTask.access$600(AsyncTask.java:177)
at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:644)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:137)
at android.app.ActivityThread.main(ActivityThread.java:4745)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:511)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
at dalvik.system.NativeStart.main(Native Method)
```

GenyMotion - OS: 4.1.1 - API 16
AndroidStudio: 2.1.2
gradle: compile 'com.github.ugurcany:InfiniteListView:1.1.0'

**Comment by ugurcany** [comment]
See this: http://stackoverflow.com/q/4404177/5250273

**Comment by aitbaali** [comment]
this is happened exactly in  `infiniteListView.stopLoading();`, maybe you have to update your code.

it works great on os 6.0.1 (both Device and GenyMotion)

**Comment by aitbaali** [comment]
any update @ugurcany ?

**Comment by ugurcany** [comment]
Working on it. I will push a new release in couple of hours.

**Comment by ugurcany** [comment]
Fixed in v1.1.1

**Comment by aitbaali** [comment]
works great, Thanks
